kafka-connect-s3/pom.xml

```diff
@@ -113,6 +113,30 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>io.confluent</groupId>
+                <artifactId>kafka-connect-maven-plugin</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
+                <executions>
+                    <execution>
+                        <id>kafka-connect</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>kafka-connect</goal>
+                        </goals>
+                        <configuration>
+                            <attach>true</attach>
+                            <excludes>
+                                <exclude>org.mortbay.jetty:*</exclude>
```

Comment:
I believe these shouldn't be specified explicitly, because they are marked as provided (or transitive deps of provided deps) in the connector. This needs to be addressed in the maven plugin's code.